tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -465,6 +465,63 @@ TEST(LinalgBackendEigen, SGMatrix_cross_entropy)
 	EXPECT_NEAR(ref, result, 1e-15);
 }
 
+TEST(LinalgBackendEigen, SGMatrix_pinvh)
+{
+	float64_t A_data[] = {2.0, -1.0, 0.0,
+												-1.0, 2.0, -1.0,
+													0.0, -1.0, 2.0};
+								
+	float64_t result_data[] = {0.75, 0.5, 0.25,
```

Comment:
name this A_pinv_scipy or so, so it is clear where this comes from. A comment also will help